tests/puppeteer/campaigns/functional/BO/13_shopParameters/05_contact/04_sortContacts.js

```diff
@@ -0,0 +1,100 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ContactsPage = require('@pages/BO/shopParameters/contact/index');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParams_contact_sortContacts';
+
+let browser;
+let page;
+let numberOfContacts = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    contactsPage: new ContactsPage(page),
+  };
+};
+
+describe('Sort Contacts', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to contact page
```

Comment:
Can you describe what does the test do exactly, please ?

Reply:
Done 
I added description to first describe and a comment before tests array